common/app/views/support/HtmlCleaner.scala

```diff
@@ -785,3 +785,25 @@ case class CommercialComponentHigh(isPaidContent: Boolean, isNetworkFront: Boole
   }
 
 }
+
+object ExplainerCleaner extends HtmlCleaner {
```

Comment:
Yes, as long as they are interactives. That's why I need the specific IDs of those I want to replace.